examples/random_walk.py

```diff
@@ -0,0 +1,22 @@
+import time
+import random
+
+from ai2thor_wrapper.envs import ThorWrapperEnv
+
+if __name__ == '__main__':
+    # Random agent example with wrapper
+    max_episode_length = 10000
+    env = ThorWrapperEnv(max_episode_length=max_episode_length)
+    for episode in range(20):
+        start = time.time()
+        s = env.reset()
+        for t in range(max_episode_length):
+            a = random.randint(0, len(env.ACTION_SPACE) - 1)
```

Comment:
use random.choice(env.ACTION_SPACE)

Reply:
good call in general but i don't have a list of actions to do choice from.
`random.choice(range(env.NUM_ACTIONS))` might work though